secp256k1-sys/build.rs

```diff
@@ -26,11 +26,6 @@ extern crate cc;
 use std::env;
 
 fn main() {
-    if cfg!(feature = "external-symbols") {
-        println!("cargo:rustc-link-lib=static=secp256k1");
-        return;
-    }
-
```

Comment:
We don't want to leave `if cfg!("external-symbols")` here?

Reply:
No, all this does is set a flag that Cargo understands ... but if you are using Cargo then you shouldn't be using this flag, since you'll get symbol collisions and other messy results.